nautobot_golden_config/tests/test_models.py

```diff
@@ -133,7 +133,9 @@ def setUp(self):
     def test_absolute_url_success(self):
         """Verify that get_absolute_url() returns the expected URL."""
         url_string = self.global_settings.get_absolute_url()
-        self.assertEqual(url_string, f"/plugins/golden-config/golden-config-setting/{self.global_settings.pk}")
+        self.assertEqual(
+            url_string.rstrip("/"), f"/plugins/golden-config/golden-config-setting/{self.global_settings.pk}"
```

Comment:
why this has changed?

Reply:
https://github.com/nautobot/nautobot/pull/4034 presumably?

Reply:
@glennmatthews looks like! GC:develop has this patched, once nautobot ver is bumped the patch won't be needed anymore I guess.